### ISSUE TITLE
Update packer scripts to be compatible with more recent releases of packer

### ISSUE THREAD

**Issue by takekoputa** [darchr/gem5art-experiments#67]
As of packer 1.6.0, `iso_checksum_type` has been deprecated. The packer scripts should be updated to accommodate this change.
One example of updating this change,
```
# old script
"builder":
[{
...
"iso_checksum": "9d2b54506f8f9fdad6b72e45aff0f0de",
"iso_checksum_type": "md5",
...
}]
```
```
# new script
"builder":
[{
...
"iso_checksum": "md5:9d2b54506f8f9fdad6b72e45aff0f0de"
...
}]
```

**Comment by powerjg** [comment]
This is great! I guess all of the gem5-resources need to be updated? Maybe a Jira ticket would be useful as well.

**Comment by takekoputa** [comment]
Yes, scripts on gem5 resources should be updated. I'll make a jira issue.

Should we update the Ubuntu version as well, given that we have a newer
version? My concern is that I don't know whether parsec or gapbs work with
gcc 7.

**Comment by powerjg** [comment]
There's no need to update Ubuntu right now.
